### PR TITLE
Remove Travis, use GitHub Action

### PR DESCRIPTION
**Pull request by sidvishnoi**
Might take a few runs to make it correct.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Marked as non substantive for IPR from ash-nazg.

File: .github/workflows/pr-push.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+jobs:
+  ci:
+    name: Validate and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
```
Comment (marcoscaceres) [inline .github/workflows/pr-push.yml]: There is no LTS still, right?

**Comment by marcoscaceres** [comment]
@sidvishnoi, where do we actually set the secrets? I couldn't find it in the settings or the actions page?

File: .github/workflows/pr-push.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Run respec-validator
```
Comment (marcoscaceres) [inline .github/workflows/pr-push.yml]:
```suggestion
      - name: Do spec validation
```

File: .github/workflows/pr-push.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        run: npx respec-validator --gh-user=$GH_USER --gh-token=$GH_TOKEN index.html
+        env:
+          GH_USER: ${{ secrets.GH_USER }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      - name: Publish
```
Comment (marcoscaceres) [inline .github/workflows/pr-push.yml]:
```suggestion
      - name: Publish to /TR/
```

File: .github/workflows/pr-push.yml (hunk 1 above)
Comment (sidvishnoi) [reply .github/workflows/pr-push.yml]: Nope. Only numbers are supported presently

**Comment by sidvishnoi** [comment]
Missed review notification from this.

> where do we actually set the secrets?

https://github.com/w3c/gamepad/settings/secrets should work, given you've right permissions.

File: .github/workflows/pr-push.yml (hunk 1 above)
Comment (sidvishnoi) [reply .github/workflows/pr-push.yml]: Maybe we should use v12 now, as it'll be LTS within a month.

Comment (marcoscaceres) [reply .github/workflows/pr-push.yml]: sounds like a plan!

Comment (sidvishnoi) [inline .github/workflows/pr-push.yml]:
```suggestion
          node-version: 12
```

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
Ok, all the tokens are in place... one nit... could you put the `env` before the `run`... that way, it's more clear where things are defined and the order is more logical and matches actual execution order: the "environment is set, then things are run" and so on...

**Comment by sidvishnoi** [comment]
Close in favour of https://github.com/w3c/gamepad/pull/115

**Pull request closed without merging**